components/Search/SearchForm.tsx

```diff
@@ -176,7 +176,6 @@ export const SearchForm = ({
           {({ activeOption }) => (
             <>
               <ComboboxInput
-                id="search-field-input"
```

Comment:
<img width="1279" height="181" alt="Screenshot 2025-08-14 at 1 49 35 PM" src="https://github.com/user-attachments/assets/f395b460-9a14-4dc5-a837-4382dd8aaf09" />

I removed this id because there was a mismatch with the `for` attribute. Removing it means just using the unique id auto generated with the ComboboxInput component which removes the mismatch. Doesn't seem to have other impact. Please let me know if this id has significance.

Reply:
Thanks for confirming, @ericbakenhus! 🙏🏼